web/wp-content/plugins/marquee/inc/metaboxes.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+// @todo: Class is used for namespacing only. Use namespaces.
```

Comment:
When a class has no usage other than namespacing its variables and functions, is there a benefit of keeping it? Or should we just use PHP namespaces?